src/nat/eval/runners/red_teaming_runner/config.py

```
@@ -0,0 +1,311 @@
+                extends_key = evaluator_dict.get("_extends")
+
+                if extends_key:
+                    # Validate extends_key exists
+                    if not self.evaluator_defaults or extends_key not in self.evaluator_defaults:
+                        available = list(self.evaluator_defaults.keys()) if self.evaluator_defaults else []
+                        raise ValueError(
+                            f"Scenario '{scenario_id}' references evaluator_defaults "
+                            f"'{extends_key}' which doesn't exist. Available: {available}."
+                            f"If attempting to extend a default evaluator, make sure the required default evaluator is"
+                            "defined in the evaluator_defaults section."
+                        )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)